infrastructure/stacks/service_etl/locals.tf

```diff
@@ -36,6 +36,13 @@ locals {
   service_etl_cloudwatch_event_action          = "lambda:InvokeFunction"
   service_etl_cloudwatch_event_princinple      = "events.amazonaws.com"
 
+  service_etl_sns_name                             = "uec-sf-${var.profile}-service-etl-sns"
+  service_etl_sns_description                      = "Service Finder function to send an email if the service etl process fails"
```

Comment:
Sort of related to above, but try to avoid project specific descriptions. Again this could be part of the ticket above, or you can tweak it here if it makes sense to.